tagstudio/src/core/json_typing.py

```diff
@@ -8,6 +8,8 @@ class JsonLibary(TypedDict("", {"ts-version": str})):
     fields: list  # TODO
     macros: "list[JsonMacro]"
     entries: "list[JsonEntry]"
+    ext_list: list[str]
+    is_exclude_list: bool
     ignored_extensions: list[str]
```

Comment:
```suggestion
    ignored_extensions: NotRequired[list[str]] # deprecated
```
\+ import:
`from typing import NotRequired`

Reply:
Thanks!